backendapi/models.py

```diff
@@ -1,3 +1,17 @@
 from django.db import models
+from rest_framework import serializers
+from django.contrib.auth.models import User
+
+
+# this is the Attendant model, im unsure if we need more data? Maby date of birth? => 18 years maby?
+class Attendant(models.Model):
```

Comment:
Models should be implemented in the database repo.
We have exactly none of the identifying information listed here for attendants, but we could use some of it for the crew interface.